conversion.go

```diff
@@ -41,7 +22,11 @@ func utf16ToByte(wstr []uint16) (result []byte) {
 
 // utf16FromString creates a UTF16 char array from a string.
 func utf16FromString(str string) []uint16 {
-	return syscall.StringToUTF16(str)
+	res, err := syscall.UTF16FromString(str)
+	if err != nil {
+		return []uint16{}
```

Comment:
I guess this is a slight change in behavior. The old implementation would have panic'ed.
New solution is better.

Reply:
Yes indeed